api/package.json

```diff
@@ -89,6 +89,7 @@
 		"@directus/storage-driver-gcs": "workspace:*",
 		"@directus/storage-driver-local": "workspace:*",
 		"@directus/storage-driver-s3": "workspace:*",
+		"@directus/types": "workspace:*",
```

Comment:
is this move from `devDependencies` to `dependencies` intentional (perhaps to resolve something)?

Reply:
Answered in the updated PR post!